Source/ResponseObjects.swift

```diff
@@ -80,7 +80,9 @@ public final class S3BucketObjectList: ResponseObjectSerializable {
             let dateString = xml["LastModified"].element?.text,
             let date =  dateFromS3Date(dateString),
             let sizeString = xml["Size"].element?.text,
-            let size = Float(sizeString) else { return nil }
+            let size = Int64(sizeString) else { return nil }
+
+		print("sizeString='\(sizeString)' size=\(size) path=\(path)")
```

Comment:
Removal is in the second commit (had realized I had left this one in just after pushing the PR). Let me know if you prefer me cancelling the PR and send out a clean one.